hoprd-node-1.yaml

```diff
@@ -14,6 +14,8 @@ spec:
     enabled: true
     dnsDomain: stage.hoprtech.net
     ingressClassName: nginx
+    annotations:
+      cert-manager.io/cluster-issuer: linode-issuer
```

Comment:
There is a better option - annotations are typically deployment specific, not fixed, they should be passed in as variables loaded from values.If you the values.yaml, changing the annotations (adding, removing,...) become a matter of changing the deployment values, not the helm chart.

Reply:
I've changed to externalize it into the Helm chart values